frontend/src/mocks/msw/handlers/announcements/announcementResponse.js

```diff
@@ -0,0 +1,85 @@
+    '\n' +
+    'Phasellus maximus massa vel velit varius condimentum. Integer at ipsum nec est consequat mollis in quis orci. Maecenas vel elementum nunc. Curabitur sit amet magna lorem. Suspendisse fringilla magna ligula, at tempor odio rhoncus eget. Sed pharetra imperdiet tellus, in sodales mauris vehicula non. Fusce viverra justo vel luctus dictum. Aliquam a sem eu ante fermentum scelerisque. Proin mauris erat, consectetur vel consequat quis, porttitor eu diam. Nunc aliquam arcu in neque condimentum condimentum. Sed sed condimentum arcu.',
+  date: new Date(2022, 2, 3, 0, 0, 0),
+  comments: [],
+};
+
+const sampleAnnouncement4 = {
+  id: 3,
+  title: 'Znaleziono psa.',
+  boardName: 'Chrząszczyżewoszyce',
+  iconType: 'pets',
+  content:
+    'PiesLorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris in risus eros. Morbi euismod magna ut lacus luctus tempus. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Fusce leo eros, pretium et blandit vestibulum, accumsan nec tellus. Pellentesque odio est, posuere at lectus ac, fermentum dignissim purus. Pellentesque convallis faucibus ex sed sollicitudin. Vestibulum sodales quis felis eget facilisis. Integer venenatis, tellus in consectetur eleifend, risus nulla feugiat nisl, a ornare diam urna sed risus. Maecenas a volutpat est. Maecenas mattis odio a ligula euismod, sit amet viverra massa aliquet.\n' +
+    '\n' +
+    'Phasellus maximus massa vel velit varius condimentum. Integer at ipsum nec est consequat mollis in quis orci. Maecenas vel elementum nunc. Curabitur sit amet magna lorem. Suspendisse fringilla magna ligula, at tempor odio rhoncus eget. Sed pharetra imperdiet tellus, in sodales mauris vehicula non. Fusce viverra justo vel luctus dictum. Aliquam a sem eu ante fermentum scelerisque. Proin mauris erat, consectetur vel consequat quis, porttitor eu diam. Nunc aliquam arcu in neque condimentum condimentum. Sed sed condimentum arcu.',
+  comments: [comment4, comment5],
+  date: new Date(2022, 1, 1, 0, 0, 0),
+};
+
+const sampleAnnouncement5 = {
+  id: 4,
+  title: 'Co to za imprezka?!',
+  boardName: 'Chrząszczyżewoszyce',
+  content:
+    'PiesLorem ipsum dolor sit amearius condimenum nunc. Curabitur sit amet magna lorem. Suspendisse fringilla magna ligula, at tempor odio rhoncus eget. Sed pharetra imperdiet tellus, in sodales mauris vehicula non. Fusce viverra justo vel luctus dictum. Aliquam a sem eu ante fermentum scelerisque. Proin mauris erat, consectetur vel consequat quis, porttitor eu diam. Nunc aliquam arcu in neque condimentum condimentum. Sed sed condimentum arcu.',
+  comments: [
+    comment1,
+    comment2,
+    comment3,
+    comment4,
+    comment5,
+    comment1,
+    comment2,
+    comment3,
+    comment4,
+    comment5,
+  ],
+  date: new Date(2022, 2, 14, 0, 0, 0),
+};
+
+export const announcementsResponse = {
```

Comment:
Co oznacza announcements**Response**?
Rozumiem, że announcements to jakieś ogłoszenie, coś w stylu posta, ale response na takie ogłosznie nie mogę sobie wyobrazić.

Reply:
To jest sztuczny response serwera na zapytanie o announcements.